### PR TITLE
feat(): move drawer to root level

### PR DESCRIPTION
**Pull request by MartoYankov**

### CODE REVIEW

**Comment by MartoYankov** [comment]
Fixes https://github.com/NativeScript/template-drawer-navigation/issues/61

File: README.md [hunk 1]
```diff
@@ -10,28 +10,36 @@ App template featuring a Side Drawer component for navigation.
...
+tns create my-drawer-ng --template https://github.com/NativeScript/template-drawer-navigation-ng
 ```
 
 **NB:** Please, have in mind that the master branch may refer to dependencies that are not on NPM yet!
 
 ## Walkthrough
 
 ### Architecture
+The RadSideDrawer component is set up as an application starting point in:
+- `/app-component.ts` - sets up the side drawer content and defines a page router outlet for the pages.
```
Comment (ADjenkov) [inline README.md]: `/app-component.html` ?

File: app.component.ts [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+
+    onNavItemTap(navItemRoute: string): void {
+        this.routerExtensions.navigate([navItemRoute], {
+            transition: {
+                name: "fade"
+            }
+        });
+
+        const sideDrawer = <RadSideDrawer>app.getRootView();
+        sideDrawer.closeDrawer();
```
Comment (VladimirAmiorkov) [inline app.component.ts]: I would say that closing the drawer should be done prior doing any navigation, purely from execution point of view. But this is also ok, I ma just concerned that both actions (closing and navigation) do have animations related to the same content on the screen and may cause irregular behavior that may depend on which one is faster on different scenarios (like slower phones or different transition effects)

Comment (MartoYankov) [reply app.component.ts]: My idea was that the primary action for this tap is to navigate, so it should be done first. Closing the drawer is just a secondary action connected to UX and shouldn't stop navigation if it fails.

**Comment by dtopuzov** [comment]
run ci

**Pull request merged**